Kernel/FileSystem/FUSE/Definitions.h

```diff
@@ -0,0 +1,1152 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-2-Clause) */
```

Comment:
I am not a lawyer, but including GPLv2 code is not a good thing. Do we rely on the `OR BSD-2-Clause` statement here?
Do we need this entire file anyway?